core/lib/libtomcrypt/src/tee_ltc_provider.c

```diff
@@ -691,7 +691,7 @@ static void bn_clear(struct bignum *s)
 {
 	struct mpa_numbase_struct *bn = (struct mpa_numbase_struct *)s;
 
-	memset(bn, 0, bn->alloc);
+	memset(bn->d, 0, bn->alloc);
```

Comment:
ok Manu, agree with you. we will cleanup later the size info or value.

Maybe add this little comment to your fix:
```
-	memset(bn, 0, bn->alloc);
+	/* despite mpa_numbase_struct description, 'alloc' field a byte size */
+	memset(bn->d, 0, bn->alloc);
```

Could you rebase, add the review tag `Reviewed-by: Etienne Carriere <etienne.carriere@linaro.org>`?
thanks.